CMakeLists.txt

```diff
@@ -180,6 +180,9 @@ if (WITH_CUDA)
 endif(WITH_CUDA)
 
 find_package(PythonInterp REQUIRED)
+find_package(PythonLibs REQUIRED)
+include_directories(${PYTHON_INCLUDE_PATH})
+link_libraries(${PYTHON_LIBRARIES})
```

Comment:
Don‘t use `find_package(PythonLibs)`. There are too many situations where it picks the wrong libraries for the interpreter. Use `distutils`, as is done below.

Reply:
This issue is actually what you are seeing happen on Travis right now.